src/resolvers.ts

```diff
@@ -19,10 +19,19 @@ export const resolvers = {
 
       return foundUser;
     },
+    users: async (_, { limit }, context) => {
+      checkToken(context);
+
+      return await userRepository
+        .createQueryBuilder('user')
+        .orderBy('user.name')
+        .limit(limit || 10)
```

Comment:
só o limit não é suficiente para fazer a paginação, olhando a descrição do botinho:
1. The first 10 orders
1. 10 more orders, **skipping** 10
1. 10 more orders, **skipping** 20

aqui só daria para puxar os 10 primeiros

Reply:
Opa Matheus, essa branch é do step [1/2], que ainda não tem a paginação!